libs/ledger-live-common/src/errors.ts

```diff
@@ -88,9 +88,9 @@ export const SourceHasMultiSign = createCustomErrorClass("SourceHasMultiSign");
 export const CosmosRedelegationInProgress = createCustomErrorClass(
   "CosmosRedelegationInProgress"
 );
-export const ClaimRewardsFeesWarning = createCustomErrorClass(
-  "ClaimRewardsFeesWarning"
-);
+// export const ClaimRewardsFeesWarning = createCustomErrorClass(
```

Comment:
You can delete those lines

Reply:
Done here 69fa69ec14fa367a4245ad0aed44226137fb65ea